test_vae_training/config_current_vae.tsv

```diff
@@ -0,0 +1,14 @@
+local_dir	"/home/alexandra/Documents/Data/Common_microbes/"
```

Comment:
I'm confused. It seems like you're using the pathlib script for part 0 and this config file for part 1? Why not just do one for both?

Reply:
Good question. Unfortunately the reason there is both is because the ponyo library uses this config file whereas I've been trying to use pathlib for my new repositories. So its a little messy until I can update ponyo